src/zzz_od/gui/view/setting/setting_plugin_interface.py

```
@@ -0,0 +1,674 @@
+            btn_list=[self.import_btn, self.import_dir_btn, self.refresh_btn, self.open_dir_btn, self.open_market_btn],
+            title=gt('插件操作'),
+            content=gt('导入 zip 格式或目录格式的插件'),
+            icon=FluentIcon.APPLICATION,
+        )
+        action_group.addSettingCard(action_card)
+
+        # 说明卡片
+        self.help_card = HelpCard(
+            url='',
+            title='插件开发说明',
+            content='第三方插件需要包含 *_const.py 和 *_factory.py 文件，详见 plugins 目录下的 README.md'
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)